packages/react-static-plugin-css-modules/src/node.api.js

```diff
@@ -1,30 +1,32 @@
 import ExtractCssChunks from 'extract-css-chunks-webpack-plugin'
 
-export default ({ includePaths = [], ...rest }) => ({
+export default (options = {}) => ({
   webpack: (config, { stage }) => {
     let loaders = []
 
     const styleLoader = { loader: 'style-loader' }
     const cssLoader = {
       loader: 'css-loader',
-      options: {
+      options: Object.assign({
         modules: true,
         importLoaders: 1,
-      },
-    }
+      }, options),
+    };
 
     if (stage === 'dev') {
       // Dev
       loaders = [styleLoader, cssLoader]
     } else if (stage === 'node') {
-      cssLoader.loader = 'css-loader/locals'
+      loaders = [
+        { ...cssLoader, loader: 'css-loader/locals' },
```

Comment:
This line breaks on build (for me). The error message states `Can't resolve 'css-loader/locals'` Any ideas on what to do to debug?

Reply:
oops, might need ti npm install it. give me a few hours. (smacks forehead)

Reply:
Okay i opened a PR quick, check if that works. Im on an ipad

Reply:
Been there! I'm checking now…

Reply:
Ah this seems to rely on css-loader pre-2.0 https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#breaking-changes

Reply:
The following seems to work (no need to add dependency to package): 
```suggestion
        {
          ...cssLoader,
          loader: "css-loader",
          options: { exportOnlyLocals: true }
        },
```

Reply:
@ScriptedAlchemy Should I make a PR for this specifically or can you revise your PR once more?

Reply:
Make a PR some family stuff came up 🤰

Reply:
Ohh wow. Css loader 2 is pretty fresh correct? I just saw the major bump but never looked at it.

Reply:
Actually this is so tiny, I’ll just do it on my iPad in a hour. :) 

Note to self: RTFM

Reply:
@ScottSmith95 check my PR, will you confirm it works before we run another release. HAvent upgraded my stuff to css-loader2 yet